core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -191,26 +216,20 @@ private ApplicationAutoScalingClient createApplicationAutoScalingClient(DynamoCo
         .build();
   }
 
-  @VisibleForTesting
-  DynamoAdmin(
-      DynamoDbClient client,
-      ApplicationAutoScalingClient applicationAutoScalingClient,
-      DynamoConfig config) {
-    this.client = client;
-    this.applicationAutoScalingClient = applicationAutoScalingClient;
-    metadataNamespace = config.getTableMetadataNamespace().orElse(METADATA_NAMESPACE);
-  }
-
   @Override
-  public void createNamespace(String namespace, Map<String, String> options) {
+  public void createNamespace(String nonPrefixedNamespace, Map<String, String> options) {
     // In Dynamo DB storage, namespace will be added to table name as prefix along with dot
     // separator.
   }
 
   @Override
   public void createTable(
-      String namespace, String table, TableMetadata metadata, Map<String, String> options)
+      String nonPrefixedNamespace,
+      String table,
+      TableMetadata metadata,
+      Map<String, String> options)
       throws ExecutionException {
```

Comment:
To avoid confusion, I renamed all the `String namespace` parameters to `String nonPrefixedNamespace` for the interface methods.